### PR TITLE
Insert noarch: python in meta.yaml, fix flake8 and pre-commit error

### PR DESCRIPTION
**Pull request by bobleesj**
I was requested by @sbillinge  to make a change. 

`noarch: true` means the package is designed to be platform-independent and can be installed on any operating system that supports the package manager.

### CODE REVIEW

File: {{ cookiecutter.module_name }}/meta.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ source:
 
 build:
   preserve_egg_dir: true
+  noarch: true
```
Comment (sbillinge) [inline {{ cookiecutter.module_name }}/meta.yaml:15]: this should be `noarch: python`

**Review by sbillinge — COMMENTED** [review]
pls see inline

File: {{ cookiecutter.module_name }}/meta.yaml (hunk 1 above)
Comment (sbillinge) [reply {{ cookiecutter.module_name }}/meta.yaml:15]: confirm.   Good thinking, but this generated an error on conda-forge and I had to change it.  I was guessing the syntax.

**Review by bobleesj — COMMENTED** [review]
`noarch: python` has been added instead of `noarch: true`.

**Comment by sbillinge** [comment]
please can you fix the flake8 error and also suppress the yaml check on the meta-yaml so that we can have pre-commit passing? thx.

**Comment by bobleesj** [comment]
Like your previous commit https://github.com/Billingegroup/cookiecutter/commit/4e2864815455bccf404f142712e176b213d00a29 on `cookiecutter`, the `meta.yaml` file was excluded in the pre-commit config file.

File: {{ cookiecutter.module_name }}/meta.yaml [hunk 2]
```diff
@@ -12,6 +12,7 @@ source:
 
 build:
   preserve_egg_dir: true
+  noarch: python
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
```
Comment (bobleesj) [inline {{ cookiecutter.module_name }}/meta.yaml:18]: Added python.

File: {{ cookiecutter.module_name }}/run_test.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env python
 
 import importlib
-import sys
```
Comment (bobleesj) [inline {{ cookiecutter.module_name }}/run_test.py:4]: Fix flake8, unused.

File: test_utils.py [hunk 4]
```diff
@@ -27,7 +27,13 @@
 p.sendline("Shared utilities for diffpy packages.")
 
 p.expect("project_full_description .*")
-p.sendline("The diffpy.utils package provides functions for extracting array data from variously formatted text files and wx GUI utilities used by the PDFgui program. The package also includes an interpolation function based on the Whittaker-Shannon formula that can be used to resample a PDF or other profile function over a new grid.")
+p.sendline(
+    "The diffpy.utils package provides functions for extracting array data "
```
Comment (bobleesj) [inline test_utils.py:31]: Break into multiple lines (flake8)

**Comment by sbillinge** [comment]
Nice and green, thanks @bobleesj !

**Pull request merged**